x-pack/legacy/plugins/cloud_migration/server/routes/cluster.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import axios from 'axios';
+import { Router, RouterRouteHandler } from '../../../../server/lib/create_router';
+
+import { LocalClusterState } from '../../common/types';
+
+import { Plugins } from '../shim';
+
+const getServerGeoId = async (): Promise<{ country: string; continent: string }> => {
+  // GeoIP the server
+  const axiosOptions = {
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Routing-Key': 'super-secret',
+    },
+    validateStatus: () => true,
+  };
+  const {
+    data: { ip },
+  } = await axios.get('http://ipinfo.io', axiosOptions);
```

Comment:
Might be worth considering moving to the `core.http.fetch` service. I believe it's available as part of `HttpStart`: (https://github.com/elastic/kibana/blob/de5c452f14a5b6ad2fb05baa343889ac36226cdc/src/core/public/http/types.ts#L36) but if it isn't, you could use `node-fetch` to start with and then migrate to `core.http.fetch` when it's available.